pegasus_wrapper/workflow.py

```diff
@@ -180,6 +208,8 @@ def run_python_on_parameters(
         category: Optional[str] = None,
         use_pypy: bool = False,
         container: Optional[Container] = None,
+        pre_job_bash: str = "",
```

Comment:
Not sure I fully understand the purpose of the pre- and post-bash stuff - how are you imagining these being used? I'm just thinking that if I want to print stuff out, I can do it via the work I'm invoking and if it's pre- or post-processing then those belong as their own jobs in the DAG.

Reply:
I need that for working with the Machine Translation CNN repository as the train scripts from there expect the `PYTHONPATH` to be the root of the repo so I needed a way to add that into the generate bash script. Example usage: `pre_job_bash=f"export PYTHONPATH={cnn_root}",`

I agree that a user shouldn't do any pre or post-processing in the bash script, I can make that clear in the documentation.

Reply:
Are there any *other* scenarios you can think of in which you would need that? If so, then adding in the example and making it more clear in the documentation is good. If not though, what do you think about just having an option for `set_pythonpath_to_base`?

Reply:
Hmm, I implemented this way in case there was the need to export more than just a `PYTHONPATH` but I can't think of any other specific use cases at this moment.

Reply:
Okay, I defer to your judgement in this case. Either option seems fine. The current implementation is definitely more flexible.